dom.bs

```diff
@@ -761,15 +764,14 @@ initialized to false.
 the user agent to dispatch an <a>event</a> whose {{Event/isTrusted}} attribute is initialized to
 false.
 
-The <dfn attribute for=Event><code>timeStamp</code></dfn> attribute
-must return the value it was initialized to. When an
-<a>event</a> is created the attribute must be
-initialized to the number of milliseconds that have passed since
-00:00:00 UTC on 1 January 1970, ignoring leap seconds.
-<!-- leap seconds are ignored by JavaScript too -->
+The <dfn attribute for=Event><code>timeStamp</code></dfn> attribute must return the value it was
+initialized to. If available, the attribute should be initialized with the high resolution time of
+the occurrence that the event is signaling, for example the moment when a particular user
+interaction occurred. Otherwise it should be initialized with the creation time of the
+<var>event</var>.
```

Comment:
Done. The wording now says ```When an <a>event</a> is created the attribute must be initialized with a {{DOMHighResTimeStamp}} representing the high resolution time in milliseconds that has passed since the <a>time origin</a>.``` 

This is similar to previous text but with new time type and origin. This initial value may change in event creation algorithm which now have the wording about occurrence time etc.